Tasks/PyPIPublisher/publisher.ts

```diff
@@ -0,0 +1,41 @@
+var tl = require('vsts-task-lib');
+var os = require('os');
+var path = require('path');
+var util = require("util");
+var workingDirectory = tl.getInput('wd', true);
+var serviceEndpointId = tl.getInput('serviceEndpoint', true);
+var homedir = os.homedir();
+var pypircFilePath = path.join(homedir, ".pypirc");
+var setupFilePath = path.join(workingDirectory, "setup.py");
+var distDirectoryPath = path.join(path.normalize(path.join(workingDirectory, "..")), "dist")+"/*";
```

Comment:
not good idea to use sibling folder of working dir. check if sdist takes a output path and then use some path like temp